<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">PyMOL Demo Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>86</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #000;color:white; padding: 20px</string>
     </property>
     <property name="text">
      <string notr="true">Three-Arm Turnstile Assistant Plugin
By Yunwen Tao</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="turnstile">
     <item row="0" column="0">
      <widget class="QPushButton" name="start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="revert">
       <property name="text">
        <string>Revert Changes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pick_finish">
       <property name="text">
        <string>Picking Finished</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="status_text">
     <property name="text">
      <string>Please specify the three-arm turnstile atoms</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="slider_angle">
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>-180°</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>180°</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="set_slider">
     <item>
      <widget class="QLineEdit" name="angle_text">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="set_angle">
       <property name="text">
        <string>Set Angle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string notr="true">Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
